SwiftGen.xcodeproj/project.pbxproj

```diff
@@ -689,6 +704,7 @@
 				C38E571320FF8F8F009C9C23 /* CoreData */,
 				DD9F687D20955DEF004EB720 /* Fonts */,
 				DDCCACD5209550F60017FB43 /* InterfaceBuilder */,
+				D4A93E25247E305200BE8651 /* JSON */,
```

Comment:
👌 I appreciate that you inserted that new folder in the middle to keep alphabetical order 👍 🙂